indexNbr_gen.js

```diff
@@ -0,0 +1,28 @@
+//タブ区切りのtxtデータを{国際地点番号 : {緯度経度(10進数)}}のJsonデータに変換する
```

Comment:
```suggestion
// 使い方
// denoの実行できる環境で以下のコマンドをターミナルで実行してください
// deno run --allow-net --allow-write indexNbr_gen.js

//タブ区切りのtxtデータを{国際地点番号 : {緯度経度(10進数)}}のJsonデータに変換する
```

使い方のコメントがあると優しいかと思います